packages/api/src/providers/tenovi.ts

```diff
@@ -39,14 +62,20 @@ export class Tenovi extends Provider {
    * Disconnects all connected devices from the user's Tenovi Gateway.
    * Removes Tenovi from the user's ProviderMap.
    *
-   * @param connectedUser The user to disconnect the device from
+   * @param connectedUser The user to disconnect the device from.
+   * @param rawParams     The raw request parameters.
    */
-  async revokeProviderAccess(connectedUser: ConnectedUser): Promise<void> {
+  async revokeProviderAccess(connectedUser: ConnectedUser, rawParams: RawParams): Promise<void> {
+    const extraParams = {
+      xTenoviApiKey: getTenoviHeaderOrFail(tenoviApiKeyPropName, rawParams),
+      xTenoviClientName: getTenoviHeaderOrFail(tenoviClientNamePropName, rawParams),
+    };
```

Comment:
If I understood correctly, this is not being used here, so we could just pass `rawParams` to `disconnectDevice` and it wouldn't need to have both `extraParams` and `rawParams`, right?

Reply:
Oh, yes! This makes it way cleaner 🙌